packages/kbn-babel-preset/styled_components_files.js

```diff
@@ -25,7 +25,6 @@ module.exports = {
     /x-pack[\/\\]solutions[\/\\]observability[\/\\]plugins[\/\\]synthetics[\/\\]/,
     /x-pack[\/\\]solutions[\/\\]observability[\/\\]plugins[\/\\]uptime[\/\\]/,
     /x-pack[\/\\]solutions[\/\\]observability[\/\\]plugins[\/\\]ux[\/\\]/,
-    /x-pack[\/\\]solutions[\/\\]security[\/\\]packages[\/\\]ecs_data_quality_dashboard[\/\\]/,
```

Comment:
this line was the puzzle piece that helped me debug what happened before with my faulty setup last time I tried to convert to emotion and failed. Turns out we have to uncomment this line to allow proper emotion compilation on webpack side. Now everything works as expected.

cc: @semd @angorayc @agusruidiazgd